earthly/flutter/installer/Earthfile

```diff
@@ -5,53 +5,44 @@ VERSION 0.8
 # Install Chrome/Chromium for testing and Chromedriver.
 # There is no Chrome-for-testing build for linux/arm64, using Chromium instead.
 # https://github.com/GoogleChromeLabs/chrome-for-testing/issues/1
-
-LET BLUE = "\033[1;34m"
-
 INSTALL_CHROME_LINUX64:
     FUNCTION
     ARG TARGETARCH
 
     # https://googlechromelabs.github.io/chrome-for-testing/
     IF [ "$TARGETARCH" = "amd64" ]
-        LET DISTR = "Debian 12 (Bookworm)"
-        LET CHROME_VERSION = 125.0.6422.76
+        LET CHROME_VERSION = 126.0.6478.61
+        LET PACKAGE_RELEASE = -1
 
-        RUN echo "${BLUE}Installing Google Chrome..."
-        # Installing dependencies for Chrome. Workaround for:
-        # https://github.com/GoogleChromeLabs/chrome-for-testing/issues/55
-        RUN chrome_deps=$(curl -s "https://raw.githubusercontent.com/chromium/chromium/${CHROME_VERSION}/chrome/installer/linux/debian/dist_package_versions.json") && \
-            deps=$(echo "$chrome_deps" | jq -r ".\"$DISTR\" | keys[]") && \
-            apt-get update && \
-            echo "$deps" | xargs apt-get install -y
-
-        # Install Chrome
-        RUN mkdir -p /opt/chrome
-        RUN curl -L "https://storage.googleapis.com/chrome-for-testing-public/${CHROME_VERSION}/linux64/chrome-linux64.zip" \
-            -o /opt/chrome/chrome.zip
-        RUN unzip -d /opt/chrome -j /opt/chrome/chrome.zip
-        RUN ln -s /opt/chrome/chrome /usr/local/bin/google-chrome
-        RUN google-chrome --version
+        # Install Google Chrome
+        RUN echo "\033[1;34mInstalling Google Chrome..."
+        # Add Google's signing key for Chrome
+        RUN curl -fSsL https://dl.google.com/linux/linux_signing_key.pub | \
+            gpg --dearmor | \
+            tee /usr/share/keyrings/google-chrome.gpg >> /dev/null
+        RUN echo deb [arch=amd64 signed-by=/usr/share/keyrings/google-chrome.gpg] https://dl.google.com/linux/chrome/deb/ stable main | \
+            tee /etc/apt/sources.list.d/google-chrome.list 
+        RUN apt-get update && apt-get install -y google-chrome-stable=${CHROME_VERSION}${PACKAGE_RELEASE}
+        RUN google-chrome --version 
 
         # Install Chromedriver
-        RUN echo "${BLUE}Installing Chromedriver..."
+        RUN echo "\033[1;34mInstalling Chromedriver..."
```

Comment:
Can you make a variable for each color? I think it's hard to interpret when embedding the color code directly on the message. It's also on several places.

Reply:
Try it already, sadly doesn't work 🥹